src/services/jwt.service.js

```diff
@@ -0,0 +1,48 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '50s',
```

Comment:
The access token expiration time is set to '50s', which is quite short for most applications. Consider increasing this duration to better suit your application's needs.